### PR TITLE
Add 'Embed OxItems Feed' Widget

### PR DESCRIPTION
**Pull request by lokothodida**
GUI to configure output_newsfeed() functionality for embedding OxItems RSS Feeds.

### CODE REVIEW

File: oxitems/js/chosen.js [hunk 1]
```diff
@@ -0,0 +1,1211 @@
...
+
+Version 1.1.0
+Full source at https://github.com/harvesthq/chosen
+Copyright (c) 2011 Harvest http://getharvest.com
+
+MIT License, https://github.com/harvesthq/chosen/blob/master/LICENSE.md
+This file is generated by `grunt build`, do not edit it by hand.
+*/
+
+(function() {
```
Comment (ghost) [inline oxitems/js/chosen.js]: Did you modify the file?

This is really a candidate for the lib folder.

File: oxitems/plugin.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// function to determine if selected element is of the type required for our dialog
```
Comment (ghost) [inline oxitems/plugin.js:1]: It's a shame that these plugin files have so much duplicate code.

File: oxitems/js/chosen.js (hunk 1 above)
Comment (lokothodida) [inline oxitems/js/chosen.js]: Moved to `oxitems/lib/chosen`

File: oxitems/plugin.js (hunk 2 above)
Comment (lokothodida) [inline oxitems/plugin.js:1]: When all of the working code is merged, we can make a subsequent release on a new branch that adds these methods to the CKEDITOR class/object and update all of the plugins to use these common methods.

**Pull request merged**